app/assets/javascripts/actions/task.js

```diff
@@ -20,19 +21,23 @@ export const toggleTaskSuccess = (task, story) => ({
 
 export const createTask = (projectId, storyId, task) => {
   return (dispatch, getState, { Task }) => {
+    dispatch((setLoading(storyId)))
     return Task.post(projectId, storyId, task)
       .then(task => {
         dispatch(createTaskSuccess(task, storyId))
-      });
+      })
+      .catch(dispatch(() => setLoading(storyId)))
```

Comment:
the action is the dispatch param.
```suggestion
      .catch(() => dispatch(setLoading(storyId)))
```